backend/pigeonpost/templates/post_detail.html

```diff
@@ -6,7 +6,7 @@
 	      <div class="col-md-8">
 	        <div class="card card-signin my-5">
 	          <div class="card-body">
-	          	<a href="{{ post.user_name.profile.get_absolute_url }}"><img src="{{ post.user_name.profile.image.url }}" class="rounded-circle" width="30" height="30" alt=""></a>
+	          	<!-- <a href="{{ post.user_name.profile.get_absolute_url }}"><img src="{{ post.user_name.profile.image.url }}" class="rounded-circle" width="30" height="30" alt=""></a> -->

```

Comment:
we still keep the code as backup?

Reply:
I would think it could be useful for a future feature, if not, we can definitely remove those lines.